lib/active_fedora/containers/container.rb

```diff
@@ -0,0 +1,28 @@
+module ActiveFedora
+  class Container < ActiveFedora::Base
```

Comment:
Maybe a comment that makes it clear that this isn't an object representation of a BasicContainer